src/typescript/01-intro-to-typescript/intro-to-typescript.md

```diff
@@ -15,6 +15,8 @@ but was passed a `number`:
 
 <a href="../static/img/typescript-error-compiling.png"><img src="../static/img/typescript-error-compiling.png" width="100%"/></a>
 
+## Why use TypeScript?
+
 ### Typechecking
 
 Typechecking is the process of verifying and enforcing the constraints of types.
```

Comment:
This could use more explanation and an example.

Come to think of it, maybe the screenshot above (or the text I recommended) should be what we show? I would still explain this a bit more and then we could use the screenshot as an example.